src/relax/transform/decompose_composite_ops.cc

```diff
@@ -110,21 +111,60 @@ class NormInferenceSimplifier : public ExprMutator {
   Map<Expr, Expr> batch_norm_map_;
 };
 
+class OpDecomposer : public ExprMutator {
+ public:
+  static Expr Decompose(Expr expr) { return OpDecomposer()(expr); }
+
+ private:
+  using ExprMutator::VisitExpr_;
+  Expr TensorToShape(const Call& call_node) {
+    ICHECK(call_node->args.size() == 1);
+    ICHECK(call_node->struct_info_.defined());
+    Expr expr = call_node->args[0];
+    const ShapeStructInfoNode* sinfo = GetStructInfoAs<ShapeStructInfoNode>(call_node);
+    ICHECK(sinfo);
+    // call builtin function that converts tensor to shape tuple
+    Var call = builder_->Emit(Call(ExternFunc("vm.builtin.tensor_to_shape"), {expr}, {},
```

Comment:
I think the Emit should be : 

```c++
static const Op& tensor_to_shape_op = Op::Get("relax.builtin.tensor_to_shape");
Var call = builder_->Emit(Call(tensor_to_shape_op, {expr}, {}, ...
```

Reply:
Thanks! Applied your suggestion.

Reply:
Thanks for the update, I am okay for that.